sw/host/opentitantool/src/command/image.rs

```diff
@@ -172,14 +204,51 @@ impl CommandDispatch for DigestCommand {
         let digest = image.compute_digest();
         if let Some(bin) = &self.bin {
             let mut file = File::create(bin)?;
-            file.write(&digest)?;
+            file.write(&digest.to_le_bytes())?;
         }
         Ok(Some(Box::new(DigestResponse {
-            digest: format!("{}", hex::encode(&digest)),
+            digest: format!("{}", hex::encode(&digest.to_be_bytes())),
```

Comment:
nit: assuming that `hex::encode` only changes the encoding we should probably have the `0x` prefix for clarity:
```suggestion
            digest: format!("0x{}", hex::encode(&digest.to_be_bytes())),
```